mappings/net/minecraft/client/particle/ParticleMobApperance.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/bnj
+CLASS none/bnj net/minecraft/client/particle/ParticleMobApperance
```

Comment:
Then my suggestion would be `ElderGuardianAppearance`. If it happens to become more generic - can always rename it! :heart: